mlrun/api/schemas/model_endpoints.py

```diff
@@ -28,36 +31,74 @@ class ModelMonitoringStoreKinds:
 
 
 class ModelEndpointMetadata(BaseModel):
-    project: Optional[str]
-    labels: Optional[dict]
-    uid: Optional[str]
+    project: Optional[str] = ""
+    labels: Optional[dict] = {}
```

Comment:
You can't assign default dictionary (or lists or any mutable objects) as it will be the SAME dictionary for all instances of the class. Clearly something we don't want. To add a default value, I know something like:

```python
import dataclasses

class ModelEndpointMetadata(BaseModel):
    labels: Optional[dict] = dataclasses.field(default_factory=lambda: {})
```
Or you can simply use the `__post_init__` method to ask if a variable is `None` then set its default value.
Try reading more about data classes and see if I remembered correctly :)